sda/host.go

```diff
@@ -92,14 +93,18 @@ func (h *Host) listen(wait bool) {
 		h.registerConnection(c)
 		h.handleConn(c)
 	}
-	go func() {
```

Comment:
I didn't test it, but do you have any idea, why this was in a go-routine prior your changes? I guess `NewHost` isn't blocking now, right?